var/httpd/htdocs/js/Core.UI.InputFields.js

```diff
@@ -247,6 +247,9 @@ Core.UI.InputFields = (function (TargetNS) {
             $('.DynamicFieldReference', $Context).each(function () {
                 Core.Agent.DynamicFieldReferenceSearch.InitElement($(this), AutoCompleteActive);
             });
+
+            //Add an extra class to Fields containing a Dynamic Field Set
+            $('.DynamicFieldSet').parent().addClass('DFSetOuterField')
```

Comment:
I would add a ", $Context" in the selector (see line 247), I think. Doesn't change anything atm, but just for accuracy.